aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -44,6 +67,24 @@ module aptos_framework::staking_config {
         voting_power_increase_limit: u64,
     }
 
+    /// Staking reward configurations that will be stored with the @aptos_framework account.
+    struct StakingRewardsConfig has copy, drop, key {
+        // The target rewards rate given out every epoch. This will be divided by the rewards rate denominator.
+        // For example, 0.001% (0.00001) can be represented as 10 / 1000000.
+        rewards_rate: u64,
```

Comment:
Instead of using rewards_rate and rewards_rate_denominator separately, let's use FixedPoint64: https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-stdlib/sources/fixed_point64.move